content/docs/reference/media/do-spaces.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Using Media with Digital Ocean Spaces
+prev: /docs/reference/media/cloudinary
+next: /docs/reference/media/s3
+---
+
+Manage **Digital Ocean Spaces media assets** in TinaCMS.
+
+## Installation
+
+### With Yarn
+
+```bash
+yarn add next-tinacms-dos
+```
+
+### With NPM
+
+```bash
+npm install next-tinacms-dos
+```
+
+## Connect with Digital Ocean Spaces
+
+You need some credentials provided by Digital Ocean Spaces to set this up properly. If you do not already have an account, you can (register here)[https://cloud.digitalocean.com/registrations/new].
+
+**next-tinacms-dos** uses environment variables within the context of a Next.js site to properly access your Digital Ocean Spaces account.
+
+Add the following variables to an `.env` file.
+
+```
+NEXT_PUBLIC_SPACES_ENDPOINT=<Your Digital Ocean Spaces Endpoint: ex. https://fra1.digitaloceanspaces.com>
+NEXT_PUBLIC_SPACES_NAME=<Your Digital Ocean Spaces Name: ex. my-spaces>
+NEXT_PUBLIC_SPACES_KEY=<Your Digital Ocean Spaces access key>
+SPACES_SECRET_KEY=<Your Digital Ocean Spaces access secret>
+```
+
+## Register the Media Store
+
+Now, you can register the Digital Ocean Spaces Media store with the instance of Tina in your app by passing the `TinaCloudDOSMediaStore` to the `TinaCMS` instance via its `mediaStore` prop.
+
+This is also where we can update our `mediaOptions` on the cms object.
+
+```tsx
+// Typically in the _app.js file of a Next.js project
+
+import dynamic from "next/dynamic";
```

Comment:
This code-block looks like it's using a dated implementation of Tina. This stuff is now configured in the schema.ts (and not the _app.tsx wrapper)
https://tina.io/docs/reference/media/overview/